ui/js/models/repository.js

```diff
@@ -204,18 +219,18 @@ treeherder.factory('ThRepositoryModel', [
 
         var loadWatchedRepos = function() {
             try {
-                return JSON.parse(sessionStorage.getItem("thWatchedRepos"));
+                return JSON.parse(localStorage.getItem("thWatchedRepos"));
             } catch (e) {
-                // sessionStorage is disabled/not supported.
+                // localStorage is disabled/not supported.
                 return {};
             }
         };
 
         var saveWatchedRepos = function() {
             try {
-                sessionStorage.setItem("thWatchedRepos", JSON.stringify(_.keys(watchedRepos)));
+                localStorage.setItem("thWatchedRepos", JSON.stringify(_.keys(watchedRepos)));
```

Comment:
We should limit this list to 3 or 4 repos.  It gets pretty long when you change a lot.